server/db/repository/AgencyRepository.js

```diff
@@ -17,7 +17,16 @@ async function createNewAgency(agencyParams) {
   }
 }
 
+async function getAllVerifiedAgencies() {
```

Comment:
shouldn't this be find({isVerified: true}) ?

Reply:
This should be true, yes 🤔